docs/getting-started/flatbuffers.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Read, write, and visualize MCAP files containing Flatbuffer data.
+---
+
+# Flatbuffers
```

Comment:
```suggestion
# FlatBuffers
```

Reply:
Add link to the official docs somewhere.

Reply:
added to read and write MCAP. I thought putting it directly under this was might be a little strange?